### DISCUSSION TITLE
target entry-point "swiper_angular" has missing dependencies:  - swiper/types

### DISCUSSION THREAD

**Discussion by fyang-architech** [nolimits4web/swiper#4838] [Q&A]
I had ngx-swiper-wrapper working in my app, but since swiper has built-in angular support, I wanted to try dropping the wrapper. installed both swiper and @types/swiper (both as dependency and devdependency) but every build gives me:

[error] Error: The target entry-point "swiper_angular" has missing dependencies:
 - swiper/types

what is up with that?

**Comment by vltansky** [comment]
Can you try with `npm install swiper@next`?
Is it SSR?
Could you share codesandbox / repo with an example?

**Reply by afreyese** [reply]
![2021-08-17 (1)](https://user-images.githubusercontent.com/69612671/130130033-c887ee75-9a72-422c-94fd-b023bc244ba8.png)
hey hello, same issue here,   "swiper": "^6.8.1", version was working fine, but 6.8.2 no, Thanks vltansky
